templates/portfolios/detail.html

```diff
@@ -31,12 +31,11 @@
     <div class="portfolio-top">
     <div class="container">
     <div class="portfolio-main">
-        <a id="new-project" onclick="show_select_portfolio_modal(); return false;" href="#" class=" btn
-        btn-success">Start a project</a>
         {% if can_change_portfolio %}
-            <a id="manage-portfolio" onclick="show_manage_portfolio_modal('{{ portfolio.id }}', '{{ portfolio.title }}'); return false;" href="#"
-               class=" btn btn-success">Manage Portfolio</a>
+            <a id="manage-portfolio" href="/portfolios/{{ portfolio.id }}/edit" class=" btn">Edit Portfolio</a>
         {% endif %}
+        <a id="new-project" onclick="show_select_portfolio_modal(); return false;" href="#" class=" btn
+        btn-success">Start a project</a>
```

Comment:
Splitting the line in the middle of the class attribute is a bit odd. Not a big deal, just slightly harder to read.